scripts/docker_run

```diff
@@ -43,6 +43,8 @@ if [[ -z "${CI:-}" ]]; then
   docker_run_flags+=('--interactive')
 fi
 
+docker_run_flags+=('--add-host=veracruz:172.17.0.1')
```

Comment:
What is this ip address? This seems to be the default subnet ip for docker. Is it actually the intention to use the subnet ip here?